### PR TITLE
Update dependencies & moment

### PR DESCRIPTION
**Pull request by arnaudforaison**
Refer to any issues

### CODE REVIEW

**Review by guillaume-chervet — DISMISSED** [review]
Nice arnaud

**Review by xballoy — DISMISSED** [review]

File: package.json [hunk 1]
```diff
@@ -76,16 +76,17 @@
...
     "jest": "^26.5.2",
-    "jest-cli": "^26.5.2",
+    "jest-cli": "^26.5.3",
     "jest-enzyme": "^7.1.2",
     "lerna": "3.22.1",
     "markdown-loader": "^6.0.0",
     "npmlog": "^4.1.2",
-    "prettier": "^1.15.3",
+    "prettier": "^2.1.2",
+    "pre-commit": "1.2.2",
```
Comment (xballoy) [inline package.json:89]: I removed pre-commit, it override husky configuration

**Review by xballoy — CHANGES_REQUESTED** [review]
Can you remove pre-commit from the package.json, it override husky configuration

**Review by xballoy — APPROVED** [review]

**Review by youf-olivier — APPROVED** [review]
Je vais bientot le virer mais au moins d'ici là ca sera a jour :D

**Pull request merged**